### PR TITLE
[docs] bevel - minikube setup

### PR DESCRIPTION
**Pull request by Mohitv**
**Changelog**
Add Minikube documentation
Fix to deploy HF fabric on minikube
 

**Reviewed by**
@jagpreetsinghsasan

 

**Linked issue**
#1796

### CODE REVIEW

File: docs/source/developer/bevel_fabric_minikube_setup.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+# Running Bevel DLT hyperledger fabric network on minikube
+
+## Pre-requisites
+
+Before proceeding, first make sure that you've completed [developer pre-requisites](./dev_prereq.md).
+
+## Setup minikube
+
+1. Please setup a Ubuntu 20.04 VM with atleast 16 GB RAM, 8 vcpu and having a public ip address, on any cloud provider like Azure, GCP etc. How to set and get a public ip address of VM, one needs to check associated cloud provider documentation. For example: [azure](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/configure-public-ip-vm)
```
Comment (jagpreetsinghsasan) [inline docs/source/developer/bevel_fabric_minikube_setup.md:14]: For the first step, one could use their own local machine to setup minikube. Can we seperate this into 2 points, 1.1) Getting public IP of local machine and 1.2) Setup on cloud and get the instance public IP

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]
Rest all looks good to me

**Comment by sownak** [comment]
@Mohitv Can the last comment be addressed please?

**Comment by Mohitv** [comment]
@sownak @jagpreetsinghsasan I have updated the feedback. Kindly check

**Review by sownak — APPROVED** [review]

**Pull request merged**